### ISSUE TITLE
Restrict a list of queues on a particular display?

### ISSUE THREAD

**Issue by burfbari** [eerotal/LibreSignage#140]
Hi,
This is a question.
Say we have 5 queues made and have 5 different display.
Is it possible to restrict (bind) a display to a particular queue?

Thank you for your efforts and your work.

**Comment by eerotal** [comment]
Yes, that's exactly how queues work. When you setup a display, you can select the queue that's displayed on that particular display.

**Comment by burfbari** [comment]
Sorry, I believe I wasn't as clear as I should as been. Let me try again, please.
I have Q1, Q2, Q3 and D1, D2 & D3.
I want to show Q1 on D1 **only**
I want to show Q2 on D2 **only**
I want to show Q3 on D3 **only**

**Comment by eerotal** [comment]
I'm not really sure if I understood correctly, but do you mean you want to ensure it's not possible to display Q1 on D2 for example? So you'd like to "block" other displays than D1 from displaying Q1? That's unfortunately not possible.

**Comment by burfbari** [comment]
Yes, you understood that correctly. Thanks for the answer. Sad! though that it is not possible.
